integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -18,6 +10,10 @@
 import org.apache.hc.core5.http.NameValuePair;
 import org.apache.hc.core5.net.URLEncodedUtils;
 
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
```

Comment:
do we have spotless configured on a project? Normally it does not allow wildcards

Reply:
added spotless for subprojects, everything is fine now